e2e-tests/src/test/resources/offer-customer-proposal-based-on-activity-event/OfferCustomerProposalBasedOnActivityEvent.json

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
The entire example scenario directory copied without changes from examples library.